docs/configuration/examples.md

```diff
@@ -0,0 +1,104 @@
+# Configuration Examples
+
+## `ignore.yml`
+
+### General
+
+```yaml
+push:
+  # No need to sync build results to the remote machine.
```

Comment:
Can we reword this into something more neutral or something? 

"No need" reads weird because it kind of implies that this configuration is not needed or something lol

Maybe "Ignore build results from local machine when pushing to remote."?